database/migrations/2014_10_12_200000_add_two_factor_columns_to_users_table.php

```diff
@@ -21,6 +22,12 @@ public function up()
             $table->text('two_factor_recovery_codes')
                     ->after('two_factor_secret')
                     ->nullable();
+
+            if (Fortify::confirmsTwoFactorAuthentication()) {
+                $table->boolean('two_factor_confirmed')
```

Comment:
Just out of convention like:  `email_verified_at` wouldn't it be better to use a timestamp instead of a boolean? This would also allow it to show when the 2fa was activated.

```suggestion
                $table->timestamp('two_factor_confirmed_at')
```

Reply:
I propose: #359